tests/integration/deployment_http_composite/test_deployment_http_composite.py

```diff
@@ -0,0 +1,139 @@
+import os
+import time
+import pytest
+
+from jina import Deployment, Executor, requests, Client, DocumentArray
+from jina.helper import random_port
+
+
+class SingleExecutorDeployment(Executor):
```

Comment:
what about `d.post(on='/bar', inputs=DocumentArray.empty(10), parameters={...})` how does it behave ?

Reply:
oh good point, will test